.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,40 @@
-- Did you search for similar issues before submitting this one?
-
-- Describe the issue you encountered:
-
-- Platform (Win7, 8, 10? macOS? Linux distro?):
-
-- Brave Version (revision SHA):
-
-- Steps to reproduce:
-    1.
-    2.
-    3.
-    
-- Actual result:
-
-- Expected result:
-
-- Will the steps above reproduce in a fresh profile? If not what other info can be added?
-
-- Is this an issue in the currently released version?
-
-- Can this issue be consistently reproduced?
-
-- Extra QA steps:
-    1.
-    2.
-    3.
-
-- Screenshot if needed:
-
-- Any related issues:
+<!--
+Have you searched for similar issues? There is a lot of feedbacks and bug reports we have received and closed as duplicate.
+Before submitting this issue, please visit our wiki for common ones: https://github.com/brave/browser-laptop/wiki
+By using search engines along with GitHub search function, you would be able to find duplicates more efficiently.
+
+For more, check out our community site: https://community.brave.com/
```

Comment:
`Our Discord community chat is available too: https://discordapp.com/invite/k57tYrS`